Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: install qa cs phpstan tests mutation-tests
+
+install:
+	composer update
```

Comment:
Should be `composer install`

Reply:
It makes sense. I totally overlooked it. Thank you for the review!